test/gtest/ucp/ucp_test.h

```diff
@@ -103,6 +103,8 @@ class ucp_test_base : public ucs::test_base {
 
         void warn_existing_eps() const;
 
+        double set_ib_ud_timeout(double timeout_sec);
```

Comment:
"setter" which returns old value is not obvious. IMO, better to add corresponding "getter". Why did you remove "reset"/"restore" method? That looked better too.

Reply:
do you think that's better to implement the following approach -
0. `test_ucp_peer_failure::m_prev_timeout = 0.;`
1. `test_ucp_peer_failure::m_prev_timeout = ucp_test::get_ib_ud_timeout();` to get the current value for futher restore (e.g. at the end of the `init()`)
2. `ucp_test::set_ib_ud_timeout(3.);`
...`// testing`
3. `test_ucp_peer_failure::restore_ib_ud_timeout();`

looks like it makes it's more complicated

Reply:
no, something one from getter with local variable or reset with `test_ucp_peer_failure::m_prev_timeout` member

Reply:
Got it, it should be something like:

> getter with local variable

1. `old_val = get_*;`
2. `set_*(3.);`
... `// testing`
4. `set_*(old_val);`

> reset with `test_ucp_peer_failure::m_prev_timeout` member

here, we still need to get the `test_ucp_peer_failure::m_prev_timeout` velue somehow before do `restore_*`

Reply:
yep